tests/wireguard/wireguard_test.go

```diff
@@ -0,0 +1,76 @@
+package test
```

Comment:
same comments :

- change function name
- I think it should be moved into testing.M function instead and have it ready before everything
- avoid using `log`. we can use `t.LogX` instead
- res should be `output`, not to be confused with the status code